third_party/terraform/website/docs/guides/getting_started.html.markdown

```diff
@@ -20,7 +20,12 @@ provider.
 
 ## Configuring the Provider
 
-First create a Terraform config file named `"main.tf"`. Inside, you'll
+First, authenticate with GCP.  The easiest way to do this is to run
```

Comment:
Can you also add some information that there's more information about authentication/credentials in a later section in this doc? We still want to encourage people to use service accounts (or ADC from a VM) since not all APIs accept gcloud or cloud shell credentials (https://github.com/terraform-providers/terraform-provider-google/issues/5388).

In that lower section, it might also be worth adding a blurb about `user_project_override` since that's directly relevant to auth, and is often a solution for people who use gcloud credentials.